### DISCUSSION TITLE
The window size when dragging the window

### DISCUSSION THREAD

**Discussion by o0oHHHo0o** [hyprwm/Hyprland#1160] [Q&A] [answered]
The window size when dragging the window is larger than the screen size. How can I fix this? In the video on the site, the window size does not change when dragging.

**Comment by vaxerski** [comment, marked as answer]
the video on the site is outdated then. When dragging, the window returns to its floating size.

**Reply by o0oHHHo0o** [reply]
Thanks for the answer:)
